tests/org.jboss.tools.openshift.cdk.server.test/src/org/jboss/tools/openshift/internal/cdk/server/test/CDKStackRuntimeTest.java

```diff
@@ -53,6 +54,10 @@
 
 	private static List<Object> results = new ArrayList<Object>();
     private static Log log = LogFactory.getLog(CDKStackRuntimeTest.class);
+    
+    private static Predicate<Integer> nonAuthenticated = code -> code == 302;
+    
+    private static Predicate<Integer> authenticated = code -> code == 301 || code == 401;
```

Comment:
isn't the opposite ? 302 is found and 401 is unauthorized..

Reply:
Good catch